src/main/java/org/sasanlabs/service/impl/EndPointsInformationProvider.java

```diff
@@ -208,11 +203,6 @@ private void addFacadeResourceInformation(
                         messageBundle.getString(description, null));
                 List<VulnerabilityType> facadeVulnerabilityTypes =
                         new ArrayList<VulnerabilityType>();
-                for (org.sasanlabs.vulnerability.types.VulnerabilityType vulnerabilityType :
```

Comment:
This was referring to the Vulnerability type but we wanted to add the Vulnerability Subtype here and also include both CWE and WASC.

Reply:
So along with this code, we need to add the CWE and WASC.

Reply:
Hi

VulnerabilityDefinition has a List of VulnerabilityType, which has value(String) and identifierType(String), So can you please explain where to include CWE and WASC?

Reply:
oh ok,

something like:
```java
facadeVulnerabilityTypes.add(
                            new VulnerabilityType("Custom", vulnerabilityType.name())
facadeVulnerabilityTypes.add(
                            new VulnerabilityType("CWE", vulnerabilityType.getCWE())
facadeVulnerabilityTypes.add(
                            new VulnerabilityType("WASC", vulnerabilityType.getWASC())
```

Reply:
Hi

3. Remove VulnerabilityType from https://github.com/SasanLabs/VulnerableApp/blob/master/src/main/java/org/sasanlabs/internal/utility/annotations/VulnerableAppRestController.java annotation

We removed VulnerabilityType[] from VulnerableAppRestController.java, on which we were iterating.

Not sure, how to proceed.

Reply:
sorry for the confusion, so these changes should be done at https://github.com/SasanLabs/VulnerableApp/pull/328/files/4a4b5cf61c0313c19dbdd6f106a7b4a2c6ee4a1e#diff-2b30db6377c09a398f894a2b738e5d797896762ee30e6a5d605f08a8416fdcceL238 place.

You need to get the Vulnerability type from the attack vector annotation and then use it.